File: allspice/Models/Favorite.cs
namespace allspice.Models
{
  public class Favorite : IRepoItem<string>
  {
    public string Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string AccountId { get; set; }
    public int RecipeId { get; set; }
  }
}

File: allspice.client/src/models/Favorite.js
export class Favorite {
  constructor(data) {
    this.id = data.Id;
    this.createdAt = data.CreatedAt;
    this.updatedAt = data.UpdatedAt;
    this.accountId = data.AccountId;
    this.recipeId = data.RecipeId;
  }
}
